### ISSUE TITLE
Не могу скомпилировать на федоре

### ISSUE THREAD

**Issue by schvabodka-man** [bcskda/2ch-cli#5]
Привет. Пытаюсь компильнуть и получаю вот это.

/usr/bin/ld: skipping incompatible /usr/lib/gcc/x86_64-redhat-linux/6.3.1/../../../libtinfo.so when searching for -ltinfo
/usr/bin/ld: skipping incompatible /lib/libtinfo.so when searching for -ltinfo
/usr/bin/ld: skipping incompatible /usr/lib/libtinfo.so when searching for -ltinfo
/usr/bin/ld: cannot find -ltinfo
collect2: error: ld returned 1 exit status
makefile:6: recipe for target 'default' failed
make: *** [default] Error 1

ЧЯДНТ?

**Comment by bcskda** [comment]
Похоже, это связано с бинарниками курсов. Посмотри тут, мб понадобится ```ncurses-compat-libs```.
https://rpmfind.net/linux/rpm2html/search.php?query=libtinfo.so.5

**Comment by bcskda** [comment]
И, возможно, тут
https://ask.fedoraproject.org/en/question/89534/parrallel-install-ncurses-6-ncurses-5-on-fedora-24/

**Comment by ghost** [comment]
Ты точно -dev-ы ncurses и curl-а поставил?
И да, если будет segmentation fault на поледнем коммите - это нормально. Его надо починить кому-то.

**Comment by bcskda** [comment]
Да, кстати, что-то вроде ```libncursesw-dev```. На днях опишу нормально зависимости.

[оффтоп]
Вообще, в первую очередь сейчас надо перевести всё на libjson, починю потом парсер HTML. Соответственно тесты все надо гонять на релизе 0.2, он хотя бы не падает (косяки в парсере те же)

**Comment by ghost** [comment]
Теперь попробуй скомпилировать 0.3.

**Comment by saber-nyan** [comment]
@schvabodka-man 
А если сейчас попробовать?
